### PR TITLE
feature(single_lidar_common_launch): load from parameter

### PR DESCRIPTION
**Pull request by vividf**
- https://github.com/autowarefoundation/autoware.universe/pull/8469
- https://github.com/autowarefoundation/autoware.universe/pull/8380
- https://github.com/autowarefoundation/autoware.universe/pull/8468
remove the default parameter, thus the launcher needs to be modified. 

related link:

- https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/103
- https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/105
- https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/104

### CODE REVIEW

**Comment by vividf** [comment]
@mojomex @beginningfan could you review this, thanks

**Comment by beginningfan** [comment]
@vividf  Thanks for your work. LGTM.
Actually, when I created this repository, I had a lot of confusion about the config of [sensor_kit](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main/sample_sensor_kit_description/config) and [individual_marams](https://github.com/autowarefoundation/autoware_individual_params/tree/main/individual_params/config/default/sample_sensor_kit). At that time, I directly used the config of [sensor_kit](https://github.com/autowarefoundation/single_lidar_sensor_kit_launch/tree/main/single_lidar_sensor_kit_description/config). However, now [autoware.launch.xml](https://github.com/autowarefoundation/autoware_launch/blob/8699901b2f0b546ef029034e5174a2d794a46f70/autoware_launch/launch/autoware.launch.xml#L72) has always been individual_marams. If it continues to be `individual_marams`, we also need to add a default config of `single_lidar_sensor_kit` in the [autoware_individual_params](https://github.com/autowarefoundation/autoware_individual_params/tree/main/individual_params/config/default) repository to launch `single_lidar_sensor_kit` successfully. Can you please complete this task？

**Comment by cyn-liu** [comment]
@vividf Thanks for your work. LGTM. 
I will review this PR later.

File: single_lidar_common_launch/launch/nebula_node_container.launch.py
Comment (cyn-liu) [inline single_lidar_common_launch/launch/nebula_node_container.launch.py:1]: ``

Comment (cyn-liu) [inline single_lidar_common_launch/launch/nebula_node_container.launch.py:1]: please  change
```python
    nodes.append(
        ComposableNode(
            package="pointcloud_preprocessor",
            plugin="pointcloud_preprocessor::CropBoxFilterComponent",
            name="crop_box_filter_self",
```
to
```python
nodes.append(
        ComposableNode(
            package="autoware_pointcloud_preprocessor",
            plugin="autoware::pointcloud_preprocessor::CropBoxFilterComponent",
            name="crop_box_filter_self",
```
Similar modifications should also be made to other related nodes.

Comment (vividf) [reply single_lidar_common_launch/launch/nebula_node_container.launch.py:1]: In vscode, the layout looks fine. 
image

Also, I didn't modify this part

Comment (vividf) [reply single_lidar_common_launch/launch/nebula_node_container.launch.py:1]: The layout in vscode looks fine
image


Also, I didn't modify this part of the code

**Comment by vividf** [comment]
@cyn-liu If everything works well, please merge this PR :) 
Thanks!

File: single_lidar_common_launch/launch/nebula_node_container.launch.py
Comment (cyn-liu) [reply single_lidar_common_launch/launch/nebula_node_container.launch.py:1]: When I update to the latest Autoware, there will be an error when compiling that there is no package pointcloud_preprocessor, You must modify it to autoware_pointcloud_preprocessor.

Comment (vividf) [reply single_lidar_common_launch/launch/nebula_node_container.launch.py:1]: Ah, thanks, I didn't see the changes you mentioned. 
I thought it was a layout issue. 

Thanks!

Comment (vividf) [reply single_lidar_common_launch/launch/nebula_node_container.launch.py:1]: @cyn-liu these are fixed in https://github.com/autowarefoundation/single_lidar_sensor_kit_launch/pull/3/commits/1e2573df3d08f6a7ed6fb450c0ea4648a053a209

**Comment by cyn-liu** [comment]
> @cyn-liu If everything works well, please merge this PR :) Thanks!

when I launch autoware with  param `sensor_moel:=single_lidar_sensor_kit`, I will get the following errors:
Reproduce steps:
1. Modify file `logging_simulator.launch.xml` and set system, localization, perception, planning, control to false
2. Modify file `single_lidar_sensor_kit_launch/single_lidar_common_launch/launch/nebula_node_container.launch.py` and 
annotate [line104 ~ line140](https://github.com/autowarefoundation/single_lidar_sensor_kit_launch/blob/ce550a82e220c36cf6af8aa9356d0da274968e35/single_lidar_common_launch/launch/nebula_node_container.launch.py#L104) 
3. Modify [input](https://github.com/autowarefoundation/single_lidar_sensor_kit_launch/blob/ce550a82e220c36cf6af8aa9356d0da274968e35/single_lidar_common_launch/launch/nebula_node_container.launch.py#L159) to point cloud topic name in your own bag.

```shell
# ros2 launch autoware_launch logging_simulator.launch.xml map_path:=./banqiao-new/ sensor_model:=single_lidar_sensor_kit
[INFO] [launch]: All log files can be found below /root/.ros/log/2024-11-11-06-16-28-844356-cynthia-liu-9617
[INFO] [launch]: Default logging verbosity is set to INFO
^C[WARNING] [launch]: user interrupted with ctrl-c (SIGINT)
root@cynthia-liu:/home/lc/workspace/humble/autoware_20241023# 
root@cynthia-liu:/home/lc/workspace/humble/autoware_20241023# 
root@cynthia-liu:/home/lc/workspace/humble/autoware_20241023# ros2 launch autoware_launch logging_simulator.launch.xml map_path:=./banqiao-new/ sensor_model:=single_lidar_sensor_kit
[INFO] [launch]: All log files can be found below /root/.ros/log/2024-11-11-06-16-46-316441-cynthia-liu-9618
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [component_container_mt-1]: process started with pid [9635]
[INFO] [robot_state_publisher-2]: process started with pid [9637]
[INFO] [component_container_mt-3]: process started with pid [9639]
[INFO] [map_hash_generator-4]: process started with pid [9641]
[INFO] [autoware_map_projection_loader_node-5]: process started with pid [9643]
[INFO] [component_container_mt-6]: process started with pid [9645]
[INFO] [vehicle_velocity_converter_node-7]: process started with pid [9647]
[INFO] [component_container_mt-8]: process started with pid [9649]
[INFO] [web_server.py-9]: process started with pid [9651]
[INFO] [initial_pose_adaptor_node-10]: process started with pid [9653]
[INFO] [routing_adaptor_node-11]: process started with pid [9655]
[INFO] [component_container_mt-12]: process started with pid [9657]
[INFO] [rviz2-13]: process started with pid [9659]
[robot_state_publisher-2] [INFO] [1731305817.038982266] [robot_state_publisher]: got segment base_link
[robot_state_publisher-2] [INFO] [1731305817.039037120] [robot_state_publisher]: got segment sensor_kit_base_link
[robot_state_publisher-2] [INFO] [1731305817.039043219] [robot_state_publisher]: got segment velodyne_top
[robot_state_publisher-2] [INFO] [1731305817.039047122] [robot_state_publisher]: got segment velodyne_top_base_link
[autoware_map_projection_loader_node-5] Load ./banqiao-new//lanelet2_map.osm
[autoware_map_projection_loader_node-5] DEPRECATED WARNING: Loading map projection info from lanelet2 map may soon be deleted. Please use map_projector_info.yaml instead. For more info, visit https://github.com/autowarefoundation/autoware.universe/blob/main/map/map_projection_loader/README.md
[rviz2-13] QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
[component_container_mt-6] [INFO] [1731305817.106346337] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Load Library: /home/lc/workspace/humble/autoware_20241023/install/glog_component/lib/libglog_component.so
[component_container_mt-6] [INFO] [1731305817.108693660] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<GlogComponent>
[component_container_mt-6] [INFO] [1731305817.108719472] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<GlogComponent>
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/sensing/lidar/top/glog_component' in container '/sensing/lidar/top/autoware_pointcloud_preprocessor/pointcloud_container'
[component_container_mt-6] [INFO] [1731305817.113980571] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Load Library: /home/lc/workspace/humble/autoware_20241023/install/autoware_pointcloud_preprocessor/lib/libpointcloud_preprocessor_filter.so
[component_container_mt-3] [INFO] [1731305817.184102713] [map.map_container]: Load Library: /home/lc/workspace/humble/autoware_20241023/install/map_loader/lib/libpointcloud_map_loader_node.so
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/autoware_state' in container '/adapi/container'
[component_container_mt-1] [INFO] [1731305817.191027095] [pointcloud_container]: Load Library: /home/lc/workspace/humble/autoware_20241023/install/glog_component/lib/libglog_component.so
[component_container_mt-1] [INFO] [1731305817.194095479] [pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<GlogComponent>
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/diagnostics' in container '/adapi/container'
[component_container_mt-1] [INFO] [1731305817.194126254] [pointcloud_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<GlogComponent>
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/pointcloud_container/glog_component' in container '/pointcloud_container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/fail_safe' in container '/adapi/container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/heartbeat' in container '/adapi/container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/interface' in container '/adapi/container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/autoware_api/external/rtc_controller/node' in container '/autoware_api/external/rtc_controller/container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/localization' in container '/adapi/container'
[web_server.py-9]  * Running on http://localhost:8888/ (Press CTRL+C to quit)
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/motion' in container '/adapi/container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/operation_mode' in container '/adapi/container'
[component_container_mt-3] [INFO] [1731305817.253348542] [map.map_container]: Found class: rclcpp_components::NodeFactoryTemplate<PointCloudMapLoaderNode>
[component_container_mt-3] [INFO] [1731305817.253386798] [map.map_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<PointCloudMapLoaderNode>
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/perception' in container '/adapi/container'
[rviz2-13] [INFO] [1731305817.259783612] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-13] [INFO] [1731305817.259902577] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/planning' in container '/adapi/container'
[rviz2-13] [INFO] [1731305817.280061064] [rviz2]: Stereo is NOT SUPPORTED
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/routing' in container '/adapi/container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/vehicle' in container '/adapi/container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/vehicle_info' in container '/adapi/container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/adapi/node/vehicle_door' in container '/adapi/container'
[component_container_mt-6] [INFO] [1731305817.340739778] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::ApproximateDownsampleFilterComponent>
[component_container_mt-6] [INFO] [1731305817.341710568] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::BlockageDiagComponent>
[component_container_mt-6] [INFO] [1731305817.342316197] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::CropBoxFilterComponent>
[component_container_mt-6] [INFO] [1731305817.342832567] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::CropBoxFilterComponent>
[component_container_mt-6] [INFO] [1731305817.356991844] [sensing.lidar.top.crop_box_filter_self]: Input and output frames are the same. Overriding has_static_tf_only to true.
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/sensing/lidar/top/crop_box_filter_self' in container '/sensing/lidar/top/autoware_pointcloud_preprocessor/pointcloud_container'
[component_container_mt-6] [INFO] [1731305817.367159292] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::ApproximateDownsampleFilterComponent>
[component_container_mt-6] [INFO] [1731305817.367200866] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::BlockageDiagComponent>
[component_container_mt-6] [INFO] [1731305817.367209684] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::CropBoxFilterComponent>
[component_container_mt-6] [INFO] [1731305817.367217599] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::CropBoxFilterComponent>
[component_container_mt-6] [INFO] [1731305817.378584112] [sensing.lidar.top.crop_box_filter_mirror]: Input and output frames are the same. Overriding has_static_tf_only to true.
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/sensing/lidar/top/crop_box_filter_mirror' in container '/sensing/lidar/top/autoware_pointcloud_preprocessor/pointcloud_container'
[component_container_mt-6] [INFO] [1731305817.384909962] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::ApproximateDownsampleFilterComponent>
[component_container_mt-6] [INFO] [1731305817.384945547] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::BlockageDiagComponent>
[component_container_mt-6] [INFO] [1731305817.384954826] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::CropBoxFilterComponent>
[component_container_mt-6] [INFO] [1731305817.384961965] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::DistortionCorrectorComponent>
[component_container_mt-6] [INFO] [1731305817.384970421] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::DistortionCorrectorComponent>
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/sensing/lidar/top/distortion_corrector_node' in container '/sensing/lidar/top/autoware_pointcloud_preprocessor/pointcloud_container'
[component_container_mt-6] [INFO] [1731305817.401081687] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::ApproximateDownsampleFilterComponent>
[component_container_mt-6] [INFO] [1731305817.401149756] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::BlockageDiagComponent>
[component_container_mt-6] [INFO] [1731305817.401159954] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::CropBoxFilterComponent>
[component_container_mt-6] [INFO] [1731305817.401168615] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::DistortionCorrectorComponent>
[component_container_mt-6] [INFO] [1731305817.401176959] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::DualReturnOutlierFilterComponent>
[component_container_mt-6] [INFO] [1731305817.401185601] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::Lanelet2MapFilterComponent>
[component_container_mt-6] [INFO] [1731305817.401193816] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::PassThroughFilterComponent>
[component_container_mt-6] [INFO] [1731305817.401201954] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::PassThroughFilterUInt16Component>
[component_container_mt-6] [INFO] [1731305817.401210397] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::PickupBasedVoxelGridDownsampleFilterComponent>
[component_container_mt-6] [INFO] [1731305817.401218808] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent>
[component_container_mt-6] [INFO] [1731305817.401226857] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::PointCloudConcatenationComponent>
[component_container_mt-6] [INFO] [1731305817.401234925] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::PointCloudDataSynchronizerComponent>
[component_container_mt-6] [INFO] [1731305817.401243036] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::PointcloudAccumulatorComponent>
[component_container_mt-6] [INFO] [1731305817.401251119] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::PolygonRemoverComponent>
[component_container_mt-6] [INFO] [1731305817.401259268] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::RadiusSearch2DOutlierFilterComponent>
[component_container_mt-6] [INFO] [1731305817.401267624] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::RandomDownsampleFilterComponent>
[component_container_mt-6] [INFO] [1731305817.401276822] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::RingOutlierFilterComponent>
[component_container_mt-6] [INFO] [1731305817.401284976] [sensing.lidar.top.autoware_pointcloud_preprocessor.pointcloud_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<autoware::pointcloud_preprocessor::RingOutlierFilterComponent>
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/sensing/lidar/top/ring_outlier_filter' in container '/sensing/lidar/top/autoware_pointcloud_preprocessor/pointcloud_container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/map/pointcloud_map_loader' in container '/map/map_container'
[component_container_mt-3] [INFO] [1731305817.482563072] [map.map_container]: Load Library: /home/lc/workspace/humble/autoware_20241023/install/map_loader/lib/liblanelet2_map_loader_node.so
[component_container_mt-3] [INFO] [1731305817.489585326] [map.map_container]: Found class: rclcpp_components::NodeFactoryTemplate<Lanelet2MapLoaderNode>
[component_container_mt-3] [INFO] [1731305817.489608516] [map.map_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<Lanelet2MapLoaderNode>
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/map/lanelet2_map_loader' in container '/map/map_container'
[component_container_mt-3] [WARN] [1731305817.497074658] [map.lanelet2_map_loader]: ./banqiao-new//lanelet2_map.osm has no format_version(null) or non semver-style format_version(null) information
[component_container_mt-3] [INFO] [1731305817.497106526] [map.lanelet2_map_loader]: Loaded map format_version: null
[component_container_mt-3] [INFO] [1731305817.497121059] [map.map_container]: Load Library: /home/lc/workspace/humble/autoware_20241023/install/map_loader/lib/liblanelet2_map_visualization_node.so
[component_container_mt-3] [INFO] [1731305817.498238549] [map.map_container]: Found class: rclcpp_components::NodeFactoryTemplate<Lanelet2MapVisualizationNode>
[component_container_mt-3] [INFO] [1731305817.498255758] [map.map_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<Lanelet2MapVisualizationNode>
[component_container_mt-3] [INFO] [1731305817.498467535] [map.lanelet2_map_loader]: Succeeded to load lanelet2_map. Map is published.
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/map/lanelet2_map_visualization' in container '/map/map_container'
[component_container_mt-3] [INFO] [1731305817.502895969] [map.lanelet2_map_visualization]: Map is loaded
[component_container_mt-3] 
[component_container_mt-3] [INFO] [1731305817.504463453] [map.map_container]: Load Library: /home/lc/workspace/humble/autoware_20241023/install/autoware_map_tf_generator/lib/libvector_map_tf_generator_node.so
[component_container_mt-3] [INFO] [1731305817.507785989] [map.map_container]: Found class: rclcpp_components::NodeFactoryTemplate<autoware::map_tf_generator::VectorMapTFGeneratorNode>
[component_container_mt-3] [INFO] [1731305817.507808939] [map.map_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<autoware::map_tf_generator::VectorMapTFGeneratorNode>
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/map/vector_map_tf_generator' in container '/map/map_container'
[component_container_mt-3] [INFO] [1731305817.513485376] [map.vector_map_tf_generator]: broadcast static tf. map_frame:map, viewer_frame:viewer, x:275.921, y:-109.062, z:-1.89793
[rviz2-13] [INFO] [1731305818.057899785] [rviz2]: Stereo is NOT SUPPORTED
[component_container_mt-6] [ERROR] [1731305823.150393504] [sensing.lidar.top.crop_box_filter_self]: The pointcloud layout is compatible with PointXYZI. You may be using legacy code/data. Continue at your own risk
[component_container_mt-6] [WARN] [1731305823.171160995] [sensing.lidar.top.distortion_corrector_node]: Required field time stamp doesn't exist in the point cloud.
[component_container_mt-6] terminate called after throwing an instance of 'std::out_of_range'
[component_container_mt-6]   what():  vector::_M_range_check: __n (which is 5) >= this->size() (which is 4)
[component_container_mt-6] *** Aborted at 1731305823 (unix time) try "date -d @1731305823" if you are using GNU date ***
[component_container_mt-6] PC: @                0x0 (unknown)
[component_container_mt-6] *** SIGABRT (@0x25ad) received by PID 9645 (TID 0x7f53597fa640) from PID 9645; stack trace: ***
[component_container_mt-6]     @     0x7f53803a84d6 google::(anonymous namespace)::FailureSignalHandler()
[component_container_mt-6]     @     0x7f5386264520 (unknown)
[component_container_mt-6]     @     0x7f53862b89fc pthread_kill
[component_container_mt-6]     @     0x7f5386264476 raise
[component_container_mt-6]     @     0x7f538624a7f3 abort
[component_container_mt-6]     @     0x7f538650db9e (unknown)
[component_container_mt-6]     @     0x7f538651920c (unknown)
[component_container_mt-6]     @     0x7f5386519277 std::terminate()
[component_container_mt-6]     @     0x7f53865194d8 __cxa_throw
[component_container_mt-6]     @     0x7f53865104cd (unknown)
[component_container_mt-6]     @     0x7f5354641f7e autoware::pointcloud_preprocessor::RingOutlierFilterComponent::faster_filter()
[component_container_mt-6]     @     0x7f533f7471e6 autoware::pointcloud_preprocessor::Filter::faster_input_indices_callback()
[component_container_mt-6]     @     0x7f533f7667df std::_Function_handler<>::_M_invoke()
[component_container_mt-6]     @     0x7f535459dac7 _ZNSt8__detail9__variant17__gen_vtable_implINS0_12_Multi_arrayIPFNS0_21__deduce_visit_resultIvEEOZN6rclcpp23AnySubscriptionCallbackIN11sensor_msgs3msg12PointCloud2_ISaIvEEESA_E22dispatch_intra_processESt10shared_ptrIKSB_ERKNS5_11MessageInfoEEUlOT_E_RSt7variantIJSt8functionIFvRSE_EESO_IFvSP_SI_EESO_IFvRKNS5_17SerializedMessageEEESO_IFvSW_SI_EESO_IFvSt10unique_ptrISB_St14default_deleteISB_EEEESO_IFvS14_SI_EESO_IFvS11_ISU_S12_ISU_EEEESO_IFvS1A_SI_EESO_IFvSF_EESO_IFvSF_SI_EESO_IFvSD_ISV_EEESO_IFvS1J_SI_EESO_IFvRKSF_EESO_IFvS1P_SI_EESO_IFvRKS1J_EESO_IFvS1V_SI_EESO_IFvSD_ISB_EEESO_IFvS20_SI_EESO_IFvSD_ISU_EEESO_IFvS25_SI_EEEEEJEEESt16integer_sequenceImJLm8EEEE14__visit_invokeESM_S2B_
[component_container_mt-6]     @     0x7f53545d42d8 rclcpp::experimental::SubscriptionIntraProcess<>::execute_impl<>()
[component_container_mt-6]     @     0x7f53867bd11f rclcpp::Executor::execute_any_executable()
[component_container_mt-6]     @     0x7f53867c434a rclcpp::executors::MultiThreadedExecutor::run()
[component_container_mt-6]     @     0x7f5386547253 (unknown)
[component_container_mt-6]     @     0x7f53862b6ac3 (unknown)
[component_container_mt-6]     @     0x7f5386348850 (unknown)
[ERROR] [component_container_mt-6]: process has died [pid 9645, exit code -6, cmd '/opt/ros/humble/lib/rclcpp_components/component_container_mt --ros-args -r __node:=pointcloud_container -r __ns:=/sensing/lidar/top/autoware_pointcloud_preprocessor -p use_sim_time:=True -p wheel_radius:=0.383 -p wheel_width:=0.235 -p wheel_base:=2.79 -p wheel_tread:=1.64 -p front_overhang:=1.0 -p rear_overhang:=1.1 -p left_overhang:=0.128 -p right_overhang:=0.128 -p vehicle_height:=2.5 -p max_steer_angle:=0.7'].
```
Specifically, when I launch autoware with  param `sensor_moel:=sample_sensor_kit` and play the bag, the program can run normally.
You can try aunch autoware with  param `sensor_moel:=single_lidar_sensor_kit`, If every ok, I will  ignore my test results and merge this PR into the main.

**Review by cyn-liu — APPROVED** [review]

**Pull request merged**